WebDriverAgentLib/Categories/XCUIApplication+FBFocused.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <XCTest/XCTest.h>
+#import "FBElement.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface XCUIApplication (FBFocused)
+
+  /**
+   Return current focused element
+   */
+- (id<FBElement>) fb_focusedElement;
```

Comment:
Will move into `XCUIApplication+FBHelpers.h` in the next PR like https://github.com/KazuCocoa/WebDriverAgent/pull/4/files#diff-b4b112241fad0f90c09864fbe40aef25R56

Reply:
This is already implemented as `activeElement`